tests/digraph/test_strongly_connected.py

```diff
@@ -65,3 +65,55 @@ def test_number_strongly_connected_big(self):
             node = G.add_node(i)
             G.add_child(node, str(i), {})
         self.assertEqual(len(rustworkx.strongly_connected_components(G)), 200000)
+
+
+class TestCondensation(unittest.TestCase):
+    def setUp(self):
+        # グラフをセットアップ
```

Comment:
I appreciate the code comments but before submitting we'll either need to translate them to English